packages/material-ui/src/Select/Select.js

```diff
@@ -101,6 +101,8 @@ Select.propTypes = {
   classes: PropTypes.object.isRequired,
   /**
    * If `true`, the selected item is displayed even if its value is empty.
+   *
+   * If you want to show a meaningful display, the `renderValue` property must be used to check for the case of an empty selected value and provide a custom render value.
```

Comment:
Appreciate that this has been rehashed a couple of times already, but I believe this may be clearer:

```suggestion
   * In order to display a meaningful value, a function should be passed to the `renderValue` prop which returns the value to be displayed when no items are selected.
```

cc @joshwooding

Reply:
I like it :)